python/lsst/meas/extensions/scarlet/io.py

```diff
@@ -485,96 +485,141 @@ def oldScarletToData(blend: Blend, psfCenter: tuple[int, int], xy0: Point2I):
     return blendData
 
 
-class ScarletModelFormatter(TypelessFormatter):
-    """Read and write zip archives.
+def build_scarlet_model(zip_dict: dict[str, Any]) -> scl.io.ScarletModelData:
```

Comment:
This doesn't feel like a public function to me. It seems like an implementation detail. Name with leading underscore?